packages/manager/apps/dedicated/client/app/billing/autoRenew/translations/Messages_en_GB.json

```diff
@@ -121,5 +121,6 @@
   "billing_autorenew_renew_action": "Renew",
   "billing_autorenew_service_renew_requires_mean_and_date": "In order to finish enabling automatic renewal, please add a new payment method. You will then be able to select a payment date.",
   "billing_autorenew_service_activate_alert": "You have at least one service with automatic renewal, but you have not yet authorised this option.",
-  "billing_autorenew_service_type_OVH_CLOUD_CONNECT": "OVHcloud Connect"
+  "billing_autorenew_service_type_OVH_CLOUD_CONNECT": "OVHcloud Connect",
+  "billing_autorenew_service_type_ESSENTIALS": "Essentials"
```

Comment:
Why do you want to localize this? its going to be same for all languages

Reply:
The type comes from the API Ravi. And this is the way it is done for the other types...